source/Ship.h

```diff
@@ -577,6 +578,8 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	// Delays for shield generation and hull repair.
 	int shieldDelay = 0;
 	int hullDelay = 0;
+	// Recent damage taken.
+	double recentShield = 0.;
```

Comment:
`recentShieldDamage` is more clear imo

Reply:
True, often variables can be made explicit instead of requiring a comment and it's always smth that should be done. Idk how I missed it